lib/debug/frame_info.rb

```diff
@@ -118,22 +118,6 @@ def location_str
       "#{pretty_path}:#{location.lineno}"
     end
 
-    private def make_binding
-      __newb__ = self.self.instance_eval('binding')
-      self.local_variables.each{|var, val|
```

Comment:
This isn't correct as `local_variables` only yields keys but no values. But the fact that we didn't spot this probably means it's never called anyway.